tests/unit/s2n_renegotiate_test.c

```diff
@@ -436,22 +469,48 @@ int main(int argc, char *argv[])
 
         struct s2n_reneg_test_case test_cases[2000] = { 0 };
         size_t test_cases_count = 0;
+
+        /* FFDHE is very, VERY slow.
```

Comment:
How much faster does this test take now?

Reply:
On the box where I test, it was ~85s before and ~8s now.